app/scripts/background.js

```diff
@@ -65,7 +65,7 @@ const openMetamaskTabsIDs = {}
 
 // state persistence
 const diskStore = new LocalStorageStore({ storageKey: STORAGE_KEY })
-const localStore = new LocalStore()
+const localStore = new ReadOnlyNetworkStore()
```

Comment:
This needs to be conditional on some environment variable

Reply:
But I think this can be merged in to the destination branch (`e2e-2.0`) for now

Reply:
> But I think this can be merged in to the destination branch (e2e-2.0) for now

We should create an issue or some sort of hard reminder to fix this before merging the destination branch to `develop`

Reply:
Created https://github.com/MetaMask/metamask-extension/issues/6324